SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version is supported.
+[Firefox might use a slightly older version](https://hg.mozilla.org/mozilla-central/file/tip/package.json#l23), which is supported-ish.
+
+## Out Threat Model 
```

Comment:
NIt, I guess there is a typo here, right? `Out` -> `Our`

Reply:
oops, thanks.